src/webrtc/stats/transportStats.ts

```diff
@@ -0,0 +1,10 @@
+export class TransportStats {
```

Comment:
This feels like something that sould be an interface rather than a class?

Reply:
Right, I will change it 👍